.travis.yml

```diff
@@ -13,15 +13,15 @@ addons:
 
 env:
   global:
-  - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  - VERSION="12.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0" MQT_DEP="PIP"
   - COUNTRY="US"
 
   matrix:
   - LINT_CHECK="1"
   - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="database_cleanup"
-  - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="database_cleanup"
+  - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="database_cleanup" EXCLUDE="iap_alternative_provider"
```

Comment:
Why excluding the module ?

Reply:
fixed I had some issue why the module code analysis, but due to the change in the master and the rebase it's not needed anymore